docs/config-file-curations-yml.md

```diff
@@ -20,6 +20,9 @@ Curations can be used to:
   * metadata-only packages, such as Maven BOM files, do not have any source code. Thus, when the flag is set the
   _downloader_ just skips the download and the _scanner_ skips the scan. Also, any _evaluator rule_ may optionally skip
   its execution.
+* set the _is_modified_ flag:
```

Comment:
Commit message:
- Optional: Feel free to just use "docs" (the directory name) instead of "Documentation" for shortness (applies to all commits).
- s/add/Add/
- What is the "small fixes" part in this commit? I only see `is_modified` being added (which is good).

Reply:
Sorry, I don't see where I wrote "small fixes" in the commit message. 
I wrote it in the PR description to address the two commits after the first one.

Reply:
Yes, my mistake, I was looking at the PR description instead.